lib/howitzer/web/page.rb

```diff
@@ -28,7 +28,7 @@ class Page
 
       def self.inherited(subclass)
         subclass.class_eval { include Singleton }
-        PageValidator.pages << subclass
+        PageValidator.pages << subclass if subclass.validations.present?
```

Comment:
This line provides ability for user to create pages without validations. if page does not have child pages we should validate it.

Lets skip validations only for abstract pages (pages that have one or more child pages):
```ruby
if subclass.validations.present? || ObjectSpace.each_object(subclass).count.zero?
  PageValidator.pages << subclass
end
```